SwiftDungeon/GameManager.swift

```diff
@@ -43,19 +101,37 @@ class GameManager {
             UIView.animate(withDuration: 1.0, delay: 0.0, options: UIViewAnimationOptions.curveEaseIn, animations: {
                 self.scene?.view?.alpha = 1.0
             }, completion: nil)
+            self.loading = false
         })
         
         scene?.removeAllChildren()
+        enemies.removeAll()
         
         scene?.addChild(map)
         player.position = map.playerSpawn
         scene?.addChild(player)
         
         let randomIndex = Int(arc4random_uniform(UInt32(map.enemySpawns.count)))
-        let pTemp = Player()
+        let pTemp = Enemy()
         pTemp.position = map.enemySpawns[randomIndex]
         pTemp.setScale(5)
         scene?.addChild(pTemp)
+        enemies.append(pTemp)
+        
+        let randomIndex2 = Int(arc4random_uniform(UInt32(map.enemySpawns.count)))
+        let pTemp2 = Enemy()
+        pTemp2.position = map.enemySpawns[randomIndex2]
+        pTemp2.setScale(5)
+        scene?.addChild(pTemp2)
+        enemies.append(pTemp2)
+        
+        let randomIndex3 = Int(arc4random_uniform(UInt32(map.enemySpawns.count)))
+        let pTemp3 = Enemy()
+        pTemp3.position = map.enemySpawns[randomIndex3]
+        pTemp3.setScale(5)
+        scene?.addChild(pTemp3)
+        enemies.append(pTemp3)
```

Comment:
Again repeated code, if you see the same code it in two places fix it, put it in a fuction.